src/components/Editable.tsx

```diff
@@ -502,6 +502,10 @@ const Editable = ({ disabled, isEditing, thoughtsRanked, contextChain, cursorOff
    */
   const onKeyDown = (e: React.KeyboardEvent) => {
     if (e.key in MODIFIER_KEYS) return
+    if (e.key === 'Enter' && duplicateThoughtsAlertTimeout.current) {
+      window.clearTimeout(duplicateThoughtsAlertTimeout.current)
+      duplicateThoughtsAlertTimeout.current = undefined
+    }
```

Comment:
This doesn't seem like the right place for this. Can you clear the alert in blur or unmount?

Reply:
Yes, you're right